src/styles/index.scss

```diff
@@ -2,2 +1,61 @@
+@import './variables';
+
+.page {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $days-gap;
+  width: ($calendar-item-size + $calendar-gap) * $calendar-item-columns;
```

Comment:
The expression `($calendar-item-size + $calendar-gap) * $calendar-item-columns` uses variables `$calendar-item-size`, `$calendar-gap`, and `$calendar-item-columns`. Make sure these variables are defined in the `variables` file or elsewhere in your code.